Update/some_004.txt

```diff
@@ -3416,7 +3416,7 @@ void main()
 //r教室の出口の影に、じっと佇む一つの影が見える。誰だろうと思って近づいていくと、…千紗登ちゃんがすっ、と姿を見せて、真っ直ぐ私に向き直った。
 	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
 	OutputLine(NULL, "教室の出口の影に、じっと佇む一つの影が見える。誰だろうと思って近づいていくと、…千紗登ちゃんがすっ、と姿を見せて、真っ直ぐ私に向き直った。",
-			NULL, "I could see a single shadow standing in the exit of the classroom. I thought about who it could be while I approached... Then Chisato-chan came out and turned straight toward me.", Line_Normal);
+			NULL, "I could see a silhouette transfixed at the door to the classroom. As I approached thinking about who it could be ... I saw it was Chisato-chan, who swiftly turned around to face me.", Line_Normal);
```

Comment:
```suggestion
			NULL, "I could see a silhouette transfixed at the door to the classroom. As I approached, thinking about who it could be ... I saw it was Chisato-chan, who swiftly turned around to face me.", Line_Normal);
```